dspace-api/src/test/data/dspaceFolder/config/entities/relationship-types.xml

```diff
@@ -109,4 +109,18 @@
             <min>0</min>
         </rightCardinality>
     </type>
+    <type>
+        <leftType>JournalIssue</leftType>
+        <rightType>Publication</rightType>
+        <leftLabel>isPublicationOfJournalIssue</leftLabel>
+        <rightLabel>isJournalIssueOfPublication</rightLabel>
+        <leftCardinality>
+            <min>0</min>
+            <max>2147483647</max>
```

Comment:
In configuration, do we really have to configure `<max>` to be the maximum allowable integer? Is there a way to either leave out the `<max>` or set it to `-1` or `null`, so that you don't have to remember to set it to the maximum integer?

Reply:
This has been adjusted in the config to be in line with https://github.com/DSpace/DSpace/blob/configurable_entities/dspace-api/src/test/data/dspaceFolder/config/entities/relationship-types.xml#L9
It was indeed not supposed to contain the max integer